### DISCUSSION TITLE
求助！存档消失

### DISCUSSION THREAD

**Discussion by Aofeng7** [Meloong-Git/PCL#4353] [提问] [answered]
求助！所有存档消失。进入pcl后之前的主题被换为默认主题，许多设置变为默认，版本隔离关闭（一直开启没有手动关闭过）。打开游戏（1.21）发现mod版本错误。我下载了1.20.1和1.21两个版本，1.21存档和mod文件夹变为1.20.1中的文件。原1.21中的所有存档和mod消失.

**Comment by Pigeon0v0** [comment, marked as answer]
你可以用 EveryThing 查找一下目录看看？

**Comment by MoYuan-CN** [comment]
PCL 主题更换与设置变成默认不可能是 PCL 自己做的，只有可能是你的注册表和/或PCL文件夹内的配置丢失了……

**Comment by Aofeng7** [comment]
好的谢谢，找到存档了
